js/privatebin.js

```diff
@@ -2257,7 +2257,11 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             password = $passwordDecrypt.val();
 
             // hide modal
-            $passwordModal.modal('hide');
+            if (typeof bootstrap !== 'undefined' && bootstrap.Tooltip.VERSION) {
+                (new bootstrap.Modal($passwordModal[0])).hide();
```

Comment:
It seems this isn't working. Maybe we could use `bootstrap.Modal.getInstance()`.

Reply:
The interface between bootstrap 3 and 5 has changed. The above should detect and trigger if version 5 is detected (bootstrap5 template, not yet the default).

Reply:
In Bootstrap 5, when you use `new` to get a modal instance, it is reinitialized and attributes like `_isShown` will be [set to false](https://github.com/twbs/bootstrap/blob/6e1f75f420f68e1d52733b8e407fc7c3766c9dba/js/src/modal.js#L73). So, when `hide()` is called, it [returns](https://github.com/twbs/bootstrap/blob/6e1f75f420f68e1d52733b8e407fc7c3766c9dba/js/src/modal.js#L125) without actually hiding the modal.

Reply:
Thank you again, I was a bit confused by that part of the bootstrap 5.3 docs, now it makes more sense. I've updated the code to store the created object in the local context to be able to reuse it. Also found out why the focus of password field didn't work anymore, the data attributes in b5 are called slightly differently.